contracts/RibbonStakingRewards.sol

```diff
@@ -175,7 +175,7 @@ contract StakingRewards is
         if (time < startEmission) {
             return 0;
         }
-        return time.sub(startEmission).div(1 weeks);
+        return time.sub(startEmission).div(1 weeks).add(1);
```

Comment:
I shifted it by a week. Makes more sense this way. Otherwise people staking first week after startEmission are as if they staked right beforehand because a whole week had not come up yet